### DISCUSSION TITLE
Welcome to The-OldOS-Project Discussions!

### DISCUSSION THREAD

**Discussion by zzanehip** [zzanehip/The-OldOS-Project#66] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ThatWindowsFan** [comment]
can you make a Windows version?

My OS: Windows 11 Home

**Comment by lgrachov** [comment]
No

**Comment by TT0FF999** [comment]
Hi g can u share the Message.swift code with us by making it usable pls 

and my last question and can I make the project compatible with swiftplaygrounds on iPadOS 26? 
• Or it requires several resources and packages and which are not supported on this hardware

" my pc and under Windows and even with a virtual machine under macos26 and xcode26 I can't launch the simulator that's why I try with {swiftplaygrounds} "

**Reply by TT0FF999** [reply]
I've just that bruh then how it's impossible fr ![image](https://github.com/user-attachments/assets/4a759516-1314-432a-9ae2-815c67aec677)

**Reply by TT0FF999** [reply]
![image](https://github.com/user-attachments/assets/16651402-80f6-440e-bbb3-bb425e0a1e60)

**Reply by TT0FF999** [reply]
By the way I completely succeeded but I mainly use a twilio api so it’s not perfect but largely functional ![image](https://github.com/user-attachments/assets/e7f7787e-8e35-4407-b90f-c748d83c4ceb)

**Reply by TT0FF999** [reply]
![image](https://github.com/user-attachments/assets/f2162f11-4bf8-4f06-95c3-8b6c4944845b)

**Comment by penguenkaciran78** [comment]
Hi! According to the information on TestFlight, the app was supposed to launch last Saturday, but it hasn’t been released yet. Has the launch been delayed? Thanks for this amazing project btw
